src/components/menu/index.js

```diff
@@ -16,7 +16,7 @@ export const Menu = ({ switchTab }) => {
     <div className="hamburger-menu">
       <input className="menuToggle" type="checkbox" />
       <button className="menu__btn" htmlFor="menuToggle" onClick={handleClick}>
-        <span></span>
+        <img src={icon} alt=""></img>
```

Comment:
There's no need to set a close img tag. You can put it like this: 

`<img />`

Also define the alt attribute, it's not a good practice to leave it empty